pkl-cli/src/main/kotlin/org/pkl/cli/commands/LSPCommand.kt

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright © 2024 Apple Inc. and the Pkl project authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.pkl.cli.commands
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.options.flag
+import com.github.ajalt.clikt.parameters.options.option
+import org.pkl.commons.cli.commands.single
+import org.pkl.lsp.PklLSP
+
+class LSPCommand(helpLink: String) :
```

Comment:
nit: `LspCommand` would be more consistent with the rest of the codebase (`HttpClient`, `IoUtils`, `ExecutorSpi`, etc.)